src/System.Diagnostics.TraceSource/tests/TraceClassTests.cs

```diff
@@ -341,9 +351,42 @@ public void TraceTest01()
         }
 
         [Fact]
```

Comment:
FrameworkDescription is only for diagnostic purposes it is not intended to be used for runtime decisions.